src/main/java/team/themoment/hellogsmv3/domain/member/controller/MemberController.java

```diff
@@ -2,17 +2,35 @@
 
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.*;
-import team.themoment.hellogsmv3.domain.member.dto.FoundMemberResDto;
+import team.themoment.hellogsmv3.domain.member.dto.request.GenerateCodeReqDto;
+import team.themoment.hellogsmv3.domain.member.dto.response.FoundMemberResDto;
 import team.themoment.hellogsmv3.domain.member.service.QueryMemberByIdService;
+import team.themoment.hellogsmv3.domain.member.service.impl.GenerateCodeServiceImpl;
+import team.themoment.hellogsmv3.domain.member.service.impl.GenerateTestCodeServiceImpl;
 import team.themoment.hellogsmv3.global.common.handler.annotation.AuthRequest;
+import team.themoment.hellogsmv3.global.common.response.CommonApiResponse;
 
 @RestController
 @RequestMapping("/member/v3")
 @RequiredArgsConstructor
 public class MemberController {
 
+    private final GenerateCodeServiceImpl generateCodeService;
+    private final GenerateTestCodeServiceImpl generateTestCodeService;
     private final QueryMemberByIdService queryMemberByIdService;
 
+    @PostMapping("/member/me/send-code")
+    public CommonApiResponse sendCode(@AuthRequest Long memberId, @RequestBody GenerateCodeReqDto reqDto) {
+        generateCodeService.execute(memberId, reqDto);
+        return CommonApiResponse.success("전송되었습니다.");
+    }
+
```

Comment:
```suggestion
    @PostMapping("/member/me/send-code")
    public CommonApiResponse sendCode(
        @AuthRequest Long memberId, 
        @RequestBody GenerateCodeReqDto reqDto
    ) {
        generateCodeService.execute(memberId, reqDto);
        return CommonApiResponse.success("전송되었습니다.");
    }

```

위와 같은 format이 가독성이 더 좋은 거 같아요.

Reply:
넵 suggestion 반영했습니다! 감사합니다